survey/settings.py

```diff
@@ -78,18 +76,27 @@
 
 WSGI_APPLICATION = 'survey.wsgi.application'
 
-
 # Database
 # https://docs.djangoproject.com/en/4.2/ref/settings/#databases
+#
+# DATABASES = {
+#     'default': {
+#         'ENGINE': 'django.db.backends.sqlite3',
+#         'NAME': BASE_DIR / 'db.sqlite3',
+#     }
+# }
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.postgresql',
```

Comment:
It's OK to push these values to Github as this is a learning project. But in real project, we often user environment variables @newDaria

Reply:
ok